iib/workers/tasks/fbc_utils.py

```diff
@@ -1,7 +1,12 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 # This file contains functions that are common for File-Based Catalog image type
+import os
 
-from iib.workers.tasks.utils import skopeo_inspect
+from iib.exceptions import IIBError
+from iib.workers.config import get_worker_config
+
+# path to index.db in our temp directories used in IIB code
+TEMP_INDEX_DB_PATH = 'database/index.db'
```

Comment:
would it be better to add it to config.py?

Reply:
I was thinking about it.

Reply:
I guess we can put it there but I would like to have some place more as constants then "config". So I was not sure if to put it there or not. This should not be changed (configured). That is why I put it here...

Reply:
we are having same path in https://github.com/release-engineering/iib/pull/323/files#diff-1bece9798c37acfe1ca705657d12580325f65a8edd965d6c3d004c7fa3237b40R74, could we reuse the variable? in that case we would like to have it a common location

Reply:
@lipoja Can I please ask why do you not want this to be configured?

Reply:
@chandwanitulsi The path should be reused already! That is why I created the constant.  :man_facepalming: and I see that I have that commit in the other PR. I will move it here. Thanks!

@yashvardhannanavati From what I see it is more constant than something to be allowed for user to change. It is used only as path where index.db is copied to in temp folder for request. I just need a place where this path can be set and used from multiple places.